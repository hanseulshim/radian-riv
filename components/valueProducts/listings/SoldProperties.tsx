import React from 'react'
import { PropertyInterface } from 'api'

interface Props {
  properties: PropertyInterface[]
}

export default function SoldProperties({ properties }: Props) {
  return (
    <div>
      {properties.map(property => {
        return <div key={property.order}>{property.address}</div>
      })}
    </div>
  )
}
