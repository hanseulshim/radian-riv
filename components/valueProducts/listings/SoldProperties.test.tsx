import * as React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import SoldProperties from './SoldProperties'

describe('SoldProperties', () => {
  let component
  beforeAll(() => {
    const props = {
      properties: []
    }
    component = shallow(<SoldProperties {...props} />)
  })
  it('should match snapshot', () => {
    expect(toJson(component)).toMatchSnapshot()
  })
})
