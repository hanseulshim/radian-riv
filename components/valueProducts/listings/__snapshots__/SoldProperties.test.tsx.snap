// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SoldProperties should match snapshot 1`] = `<div />`;
